src/bootstrap/src/core/config/tests.rs

```diff
@@ -92,7 +92,7 @@ fn detect_src_and_out() {
             //     `{build-dir}/bootstrap/debug/deps/bootstrap-c7ee91d5661e2804`
             // `{build-dir}` can be anywhere, not just in the rust project directory.
             let dep = Path::new(args.first().unwrap());
-            let expected_out = dep.ancestors().nth(4).unwrap();
+            let expected_out = dep.ancestors().nth(5).unwrap();
```

Comment:
Output path now also includes the host triple as bootstrap passes that regardless even for non-cross builds https://github.com/rust-lang/rust/blob/b2728d5426bab1d8c39709768c7e22b7f66dde5d/src/bootstrap/src/core/builder/cargo.rs#L415-L416